### PR TITLE
Use vbox cache

### PR DESCRIPTION
**Pull request by wangkuiyi**

### CODE REVIEW

**Review by Yancey0623 — DISMISSED** [review]
LGTM

File: dev.md [hunk 1]
```diff
@@ -11,11 +11,20 @@
...
+./play.sh
+```
+
+If you have a slow Internet connection to Vagrant Cloud, you might want to
+download the Ubuntu VirtualBox image manually from some mirror sites into
+`~/.cache/sqlflow/` before running the above script.
+
 ```bash
-./install.sh [inchina]
+wget -c -nv --show-progress -O downloads/ubuntu-bionic64.box \
```
Comment (lhw362950217) [inline dev.md]: -O downloads/ubuntu-bionic64.box -> -O ~/.cache/sqlflow/ubuntu-bionic64.box

and add `mkdir ~/.cache/sqlflow`

File: play.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    exit 0
+fi
+
+if [[ -z "$(vagrant plugin list | grep -o 'vagrant-disksize')" ]]; then
+    echo "Install Vagrant disk size plugin ..."
+    vagrant plugin install vagrant-disksize
+fi
+
+CACHED_BOX="$HOME/.cache/sqlflow/ubuntu-bionic64.box"
+if [[ -f $CACHED_BOX ]]; then
```
Comment (lhw362950217) [inline play.sh]: maybe we want to check if the box is present
```bash
if [[ -z "$(vagrant box list | grep 'ubuntu/bionic64')" ]]; then
    echo "Found and use cached box $CACHED_BOX"
    vagrant box add ubuntu/bionic64 $CACHED_BOX
fi
```

Comment (wangkuiyi) [reply play.sh]: Done

**Review by lhw362950217 — APPROVED** [review]
LGTM

**Pull request merged**